.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: ruby
+sudo: false
 rvm:
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.2.2
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
+  - 2.4.0
```

Comment:
well ... travis is not using latest patches :(
I did a test on one of my personal repo, specifying major.minor version and print 'ruby -v' as pre-script.
It gives me  ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-linux] for ruby 2.1.